marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/entities/entity-search-lib.sjs

```diff
@@ -365,6 +367,9 @@ function addGenericEntityProperties(result) {
     } else {
         addPrimaryKeyToResult(result, entityInstance, entityDef);
     }
+    } else {
+      console.log(`Unable to obtain a valid entity instance from document with URI '${result.uri}'; will not add entity properties to its search result`);
```

Comment:
Please change this to xdmp.log()

Reply:
ok, I didn't realize that

Reply:
@srinathgit - Its being used in few other places as well in the same file. If xdmp.log makes more sense and is more helpful then should we replace those as well ?

Reply:
console.log should be ok , there doesn't seem to be a difference